app/code/Magento/InventoryApi/Test/Api/SourceRepository/GetListTest.php

```diff
@@ -54,7 +54,7 @@ public function testGetList()
                 SearchCriteria::PAGE_SIZE => 2,
             ],
         ];
-        $expectedTotalCount = 4;
+        $expectedTotalCount = 5;
```

Comment:
Default Stock and Default Source have created in `InventoryCatalog` but `InventoryApi` module has not dependency on `InventoryCatalog`
So we could not create tests which will be green only if `InventoryCatalog` module is enabled

Reply:
I understand; what's should be a correct approach in these cases?

Reply:
I think we should not create sophisticated logic
So we can use `assertGreatherThanOrEqual`

Reply:
I already fixed